.golangci.yml

```diff
@@ -4,8 +4,6 @@ run:
 linters-settings:
   gofmt:
     simplify: false
-  govet:
-    check-shadowing: false
```

Comment:
FYI: Shadowing can still be disabled: https://golangci-lint.run/usage/linters/#govet, it's just that it can be disabled in a different way. However, the more linters we have enabled the better, so I'm fine with removing this part of config.